Source/Document Structure/SvgScript.cs

```diff
@@ -0,0 +1,81 @@
+using System.Drawing;
+using System.Drawing.Drawing2D;
+
+namespace Svg
+{
+    /// <summary>
+    /// An element used to define scripts within SVG documents.
+    /// Use the Script property to get the script content (proxies the content)
+    /// </summary>
+    [SvgElement("script")]
+    public class SvgScript : SvgElement
+    {
+
+        public string Script 
+        {
+            get { return this.Content; }
+            set { this.Content = value; }
+        }
+
+        private string _scriptType;
+
+        [SvgAttribute("type")]
+        public string ScriptType 
+        { 
+            get { return _scriptType; } 
```

Comment:
If there is no special reason, set it to `Attributes` like any other element.

Reply:
Refactored this. I think I used a wrong example to base this code on. Reworked this to use the attribute collection, this also removed the need to manually output the attributes.